packages/drivers/redux-little-router/misc/test-suite.js

```diff
@@ -0,0 +1,267 @@
+/* eslint-env jest */
+import { push } from 'redux-little-router'
+import { createStore as krmlCreateStore, when } from 'k-ramel'
+import omit from 'lodash/omit'
+
+export default (driver) => {
+  /* create k-ramel store with redux-little-router driver */
+  const createStore = (description, options, routes, impl) => {
+    const router = driver({ routes }, state => state.router, impl)
```

Comment:
Before merging @bpetetot , why don't we do something like that: 
`driver(routes, state => state.router, impl)`

Instead of `{ routes }`.

Remember that 90% of use cases should be simple to implement/write.

Reply:
Yes but in that case, I dont really like the signature of other 10% : 
```js
const router = driver(routes, state => state.router, routerForHash, routerOptions)
```
or as alternative
```js
const router = driver(undefined, state => state.router, routerForHash(routerOptions))
```

Maybe the best way to do it could be : 

```js
// 90% of use cases
const router = driver(routes, state => state.router)

// 10% of other use cases
const router = driver(routerForHash(routerOptions), state => state.router)
```

Reply:
Ok for the last snippet then :)